frontend/src/HomePage/HomePage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import axios from 'axios';
+import NavBar from '../Navbar/NavBar';
+
+class HomePage extends React.Component {
+	state = { details: [] };
+
+	componentDidMount() {
+		//let data;
+		axios
+			.get('http://localhost:8000/api/posts/')
+			.then((res) => this.setState({ details: res.data }))
+			.catch((err) => console.log(err));
+	}
+	render() {
+		return (
+			<>
+				<div>
+					<NavBar />
+				</div>
+				<div>
+					<header>(Sunposterisk) Data Generated From Django</header>
+					<hr></hr>
+					{this.state.details.map((output, id) => (
```

Comment:
```suggestion
					{posts?.map((output, id) => (
```
lastly, call our updated state